files/chef-marketplace-cookbooks/chef-marketplace/templates/default/delivery.rb.erb

```diff
@@ -12,7 +12,7 @@ delivery["chef_server"]           = "https://localhost:8443/organizations/delive
 delivery["chef_server_proxy"]     = true
 delivery["chef_username"]         = "delivery"
 delivery["primary"]               = true
-delivery["marketplace_licensing"] = true
+delivery["marketplace_licensing"] = <%= node['chef-marketplace']['platform'] != 'azure' %>
```

Comment:
Not going to keep nitpicking on this, but the marketplace licensing feature only works on AWS, right? So wouldn't we want to trigger when `== 'AWS'` or the equivalent?